cmd/dashboard/repl.js

```diff
@@ -12,9 +15,12 @@ class REPL {
 
   enhancedWriter(output) {
     if ((typeof output) === "string") {
+      this.logText.log(output);
       return output;
     } else {
-      return util.inspect(output, {colors: true});
+      const inspectedOutput = util.inspect(output, {colors: true});
+      this.logText.log(inspectedOutput);
+      return inspectedOutput;
     }
   }
```

Comment:
When running `../../bin/embark console` from the demo I have this error:
```
TypeError: Cannot read property 'log' of undefined
    at REPL.enhancedWriter (/Users/anthony/code/embark/cmd/dashboard/repl.js:22:20)
    at finish (repl.js:663:38)
    at CodeRunner.evalCode (/Users/anthony/code/embark/lib/core/modules/coderunner/codeRunner.js:87:5)
    at EventEmitter.listener (/Users/anthony/code/embark/lib/core/events.js:49:8)
    at EventEmitter.emit (events.js:182:13)
    at EventEmitter.emit (domain.js:442:20)
    at EventEmitter.request (/Users/anthony/code/embark/lib/core/events.js:43:15)
    at Console.executeCmd (/Users/anthony/code/embark/lib/modules/console/index.js:66:19)
    at EventEmitter.listener (/Users/anthony/code/embark/lib/core/events.js:49:8)
    at EventEmitter.emit (events.js:182:13)
    at EventEmitter.emit (domain.js:442:20)
    at EventEmitter.request (/Users/anthony/code/embark/lib/core/events.js:43:15)
    at REPL.enhancedEval (/Users/anthony/code/embark/cmd/dashboard/repl.js:13:17)
    at bound (domain.js:396:14)
    at REPLServer.runBound [as eval] (domain.js:409:12)
    at REPLServer.onLine (repl.js:619:10)
    at REPLServer.emit (events.js:182:13)
    at REPLServer.EventEmitter.emit (domain.js:442:20)
    at REPLServer.Interface._onLine (readline.js:290:10)
    at REPLServer.Interface._line (readline.js:638:8)
    at REPLServer.Interface._ttyWrite (readline.js:919:14)
    at REPLServer.self._ttyWrite (repl.js:692:7)
```
Also I believe that output might be displayed twice if it happens to work